models/issues/issue_project.go

```diff
@@ -48,30 +50,40 @@ func (issue *Issue) ProjectBoardID(ctx context.Context) int64 {
 }
 
 // LoadIssuesFromBoard load issues assigned to this board
-func LoadIssuesFromBoard(ctx context.Context, b *project_model.Board) (IssueList, error) {
+func LoadIssuesFromBoard(ctx context.Context, b *project_model.Board, doer *user_model.User, isClosed optional.Option[bool]) (IssueList, error) {
 	issueList := make(IssueList, 0, 10)
 
 	if b.ID > 0 {
 		issues, err := Issues(ctx, &IssuesOptions{
 			ProjectBoardID: b.ID,
 			ProjectID:      b.ProjectID,
 			SortType:       "project-column-sorting",
+			IsClosed:       isClosed,
 		})
 		if err != nil {
 			return nil, err
 		}
-		issueList = issues
+		issues, err = issues.FilterValidByDoer(ctx, doer)
```

Comment:
`IssuesOptions` has `User` option, but it has no effect when doer is nil. (no-login user)
Also maybe we can use newly added indexer.

Reply:
b2294673e83530809b958eadaa10ade4b2916bf5

Reply:
emmm, this is not correct

Reply:
import cycle will happen when using indexer 😕, so the only way is fixing the query bug.